cmd/devtools/run.go

```diff
@@ -221,3 +238,38 @@ func getFlagPathOrPanic(c *cobra.Command, flagName string, defaultValue string)
 		return defaultValue
 	}
 }
+
+// gRPCServerIsOK builds an anonymous function for use in a ProcessRunner
+// ReadyChecker callback. The anonymous function checks if the gRPC server that
+// is started by the sequencer is OK by making an HTTP request to the health
+// endpoint. Being able to connect to the gRPC server is a requirement for both
+// the Conductor and Composer services.
+func gRPCServerIsOK(envPath string) func() bool {
```

Comment:
How do you feel about renaming this to something like `getSequencerOKCallback` or similar? I think it's more accurate.